docs/topics/writing_templates.md

```diff
@@ -136,6 +136,16 @@ See [](responsive_images) for full documentation.
 
 (rich_text_filter)=
 
+## User Avatar
+
+This tag retrieves the correct avatar URL for a specified user, displaying either the user's uploaded profile image, their Gravatar image, or the default profile picture, depending on the site settings.
+
+```html+django
+{% load wagtailadmin_tags %}
+
+<img src="{% avatar_url user %}" alt="User Profile"/>
```

Comment:
As can be seen here, this tag comes from `wagtailadmin_tags` – which means it's meant for admin use only. This documentation page is about writing templates for the front-end site, so the tag shouldn't be documented here, nor should we encourage people to use it outside of the admin.

Reply:
Let's document this for admin only for now, Wagtail usually tries to limit the 'frontend' items to allow for whatever developers want to do. 

For example, it's not always true that the user image would be the same as a public facing author image. There may be different constraints, even entirely different models for an Author.

Reply:
Sounds good. I'll close this pull request now.